apps/client/src/features/rundown/event-editor/composite/EventEditorTitles.tsx

```diff
@@ -46,7 +48,10 @@ const EventEditorTitles = (props: EventEditorTitlesProps) => {
       </div>
       <div>
         <Editor.Label>Colour</Editor.Label>
-        <SwatchSelect name='colour' value={colour} handleChange={handleSubmit} />
+        <div className={style.colorSettings}>
+          <SwatchSelect name='colour' value={colour} handleChange={handleSubmit} />
+          <EventColorPicker name='colour' value={colour} icon={<IoEyedrop size={15} />} handleChange={handleSubmit} />
```

Comment:
questions: 

1. I believe the icon is at the wrong level of the abstraction.

I would think this should be included in the `EventColorPicker` element

We also dont need to inline the size for the icon, this should will come from the inherited styles or we can add it to the element styles in the CSS if necessary

2. The `Swatch` element compares the string to its constant to find out if a particular colour is the one selected. How could we write logic to know if the user has selected a custom colour and highlight the picker component? Maybe check the value against the colours array?